.gitignore

```diff
@@ -50,5 +50,4 @@ cmd/simulator/.simulator/*
 # goreleaser
 dist/
 
-# generator rpc file for e2e tests
-contract-examples/dynamic_rpc.json
\ No newline at end of file
+node_modules
```

Comment:
do we need / want this change?
I think it is from one of my PRs that was merged into this PR.
I'm ok to keep the change, just want to make sure it's intentional.

Reply:
Ya this was merged from yours. It could have been separated out, but I kept it in because it seems like a good change.